recipes/thevenin/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "thevenin" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f1c36ca017736ac485152c2816326d4cc27a745646ee32977df0d8cdcac9032
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - scipy
+    - matplotlib-base
+    - ruamel.yaml
+    - scikit-sundae
+
+test:
+  requires:
+    - python {{ python_min }}
+  imports:
+    - thevenin
+  files:
+    - basic_tests.py
+  commands:
+    - python basic_tests.py
```

Comment:
Just rename that file to run_test.py and it will run automatically during the test phase.

```suggestion
```